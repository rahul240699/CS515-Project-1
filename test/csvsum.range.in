python3 prog/csvsum.py grades.csv --col_range 3:5