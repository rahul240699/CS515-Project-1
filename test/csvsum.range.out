Test1    695.0
Test2    497.0
Test3    978.0
dtype: float64